tests/modules/controlfreec/nextflow.config

```diff
@@ -0,0 +1,23 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName:CONTROLFREEC_SOMATIC{
+        ext.args = { ["general" :[
```

Comment:
Map would look like this then

Reply:
I actually think this would need an extra explanation on how to use it and how the map fields are defined. Not sure yet how that would fit in the meta.yml